README.md

```diff
@@ -1,16 +1,50 @@
-# One Repo Split Angular (Client) & Rails (Backend) setup
+# Single repository split Angular & Ruby on Rails example
 
-An example of how you can integrate angular and rails for simple
-applications in just one repo, but without have to mix code between
-the client and the backend, or have the client code under /assets
-and try to do magic for that code to work together
+This is an example of how you can use Angular and Rails in a single repository, but
+using Gulp instead of Rails' asset pipeline to manage your interface.
 
-This is a demo only, so some concepts must be refined, but personally
-this has way less magic than making the two apps in one mix file structure/stack
+## Installation
+
+Install the dependencies:
+
+    $ npm install -g yo gulp bower
+    $ gem install rails
+
+We like [generator-gulp-angular](https://github.com/Swiip/generator-gulp-angular) to generate
+the foundation for the client, so install that:
+
+    $ npm install -g generator-gulp-angular
+
+Make a new directory for your application and `cd` into it:
+
+    $ mkdir my-application && cd $_
+
+Make a directory for your API, and another for your client:
+
+    $ mkdir api client
+
+`cd` into your client's directory and run the generator:
+
+    $ cd client
+    $ yo gulp-angular [app-name]
+
+`cd` into your API's directory and run the generator:
+
+    $ cd api
+    $ rails new [app-name]
+
+That's it! You're done.
+
+## Development
+
+Run `gulp`, then `gulp serve`:
+
+    $ gulp
+    $ gulp serve
```

Comment:
you can think of gulp as rake

then the `yo gulp-angular` will create some gulp files (from the generator) on which the tasks are defined

on this particular configuration of gulp.. the default task is `build`

that is not a dependency for the task `grunt serve`, but aparently it's a fault of the generator, since it seems to be required